### ISSUE TITLE
query on populated field

### ISSUE THREAD

**Issue by summer-liu** [florianholzapfel/express-restify-mongoose#78]
Is that possible to query on populated field? 

Say I have a collection with the schema like this:

```
var recordSchema = new Schema({
	student: {
	 type:Schema.Types.ObjectId,
	 ref: 'User'
	 }
	 .......
})
```

and in User model there is a field "school".

I want to do a query like this:
```
/records?populate=student&student.school=2340544234
```
but it says there is TypeError: Cannot read property 'paths' of undefined.

How can I do that with Express Restify Mongoose?

**Comment by PaulMougel** [comment]
Any news?

**Comment by Zertz** [comment]
This could be implemented by also accepting a JSON object for the populate parameter.

Reference: [Query conditions and other options](http://mongoosejs.com/docs/populate.html))

**Comment by Zertz** [comment]
Implemented in #154 

`/api/v1/Customers?populate={"path":"invoice","match":{"code":"ABC"}}`
